ql/cashflows/digitalcoupon.hpp

```diff
@@ -91,7 +91,7 @@ namespace QuantLib {
                       bool isPutITMIncluded = false,
                       Rate putDigitalPayoff = Null<Rate>(),
                       const ext::shared_ptr<DigitalReplication>& replication =
-                        ext::shared_ptr<DigitalReplication>(),
+                          ext::shared_ptr<DigitalReplication>(),
```

Comment:
Minor point but you can use `{}` as the default value to avoid repeating the type.

Reply:
Ok!